docker/default_config.ini

```diff
@@ -35,7 +35,7 @@ rpc-endpoint = 0.0.0.0:8090
 enable-stale-production = false
 
 # Percent of witnesses (0-99) that must be participating in order to produce blocks
-required-participation = false
+required-participation = 0
```

Comment:
The default parameter is `33`. We should change to that here.

Reply:
Sure, will do !! Thanks

Reply:
Please re-validate! Thanks a lot